platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/ReadableRosterStoreImpl.java

```diff
@@ -128,4 +134,23 @@ public Bytes getPreviousRosterHash() {
                 .filter(pair -> rosterMap.contains(new ProtoBytes(pair.activeRosterHash())))
                 .toList();
     }
+
+    /**
+     * @return the node ids from the roster entries of both active and candidate rosters combined
+     */
+    public Set<EntityNumber> getCombinedRosterEntriesNodeIds() {
+        final var activeRoster = getActiveRoster();
```

Comment:
The issue states _"When a node id is not present in any of the active rosters ..."_ which lead me to think that we'll have multiple rosters here, was that a typo in the issue description?

Reply:
I would assume it was a typo and thinking about cases where we could end up with more than one active roster..
Maybe a question for @edward-swirldslabs (I saw that in the design doc it is also written _`active rosters`_)

Reply:
Great, thanks for the clarification! Updated